cicd/forgeops-tests/lib/utils/pod.py

```diff
@@ -42,111 +37,119 @@ def product_type(self):
+        :param title: The title for the attributes.
+        :param description: Description of the data
+        :param metadata: Pod metadata
+        :param get_attributes: Keys ot attributes that are required in the dictionary.
+        :return: Metadata dictionary indexed by the attribute keys.
+        """
+        assert isinstance(title, str), 'Invalid title type %s' % type(title)
+        assert isinstance(description, str), 'Invalid descripton type %s' % type(description)
+        assert isinstance(metadata, list), 'Invalid metadata type %s' % type(metadata)
+        assert isinstance(get_attributes, set), 'Invalid get attributes type %s' % type(get_attributes)
+
+        metadata_of_interest = {'TITLE': title,
+                                'DESCRIPTION': description}
+
+        found_keys = set()
+        for line in metadata:
+            logger.debug('Reading metdata: ' + line)
+            for info_key in get_attributes:
+                if line.strip().startswith(info_key):
+                    info = line.strip().split(info_key)
+                    if len(info) == 2:
+                        metadata_of_interest[info_key] = info[1].strip(' :=')
+                        found_keys.add(info_key)
+                        logger.debug('Found ' + info_key)
+                        continue
+        assert found_keys == get_attributes
+
+        return metadata_of_interest
 
     @staticmethod
-    def is_expected_am_amster_version(am_manifest, amster_manifest):
+    def print_table(table_data):
         """
-        :param am_manifest: Manifest of version information
-        :param amster_manifest: Manifest of version information
-        Return True if the AM and Amster versions match, otherwise assert.
-        :return: True if the versions are as expected.
+        Print a table of version metadata.
+        :param table_data: Dictionary of data to be printed
         """
 
-        logger.debug('Checking AM and Amster versions match')
-        assert am_manifest['version'] == amster_manifest['version'], 'AM and Amster versions should match'
-        assert am_manifest['revision'] == amster_manifest['revision'], 'AM and Amster build revisions should match'
-        assert am_manifest['date'] == amster_manifest['date'], 'AM and Amster build date should match'
-        assert am_manifest['commons_version'] == amster_manifest['commons_version'], 'AM and Amster commons version should match'
+        assert isinstance(table_data, dict), 'Invalid title type %s' % type(table_data)
 
-        if am_manifest['jdk_version'] != amster_manifest['jdk_version']:
-            logger.warning("Different versions of java are in use.")
+        table = PrettyTable([table_data['TITLE'], table_data['DESCRIPTION']])
+        for key, value in table_data.items():
+            if key in ['TITLE', 'DESCRIPTION']:
+                continue
+            table.add_row([key, value])
+        logger.info(table)
```

Comment:
Need to add newline at beginning of this or the output looks like 
`2019-02-26 13:30:04 INFO +----------+-------------------------+

| openidm  | markg-openidm-openidm-0 |
+----------+-------------------------+
| VERSION  |      7.0.0-SNAPSHOT     |
| REVISION |         unknown         |
|   DATE   |      20190129151444     |
+----------+-------------------------+`

Reply:
This is down to the logger output.
In time this information will be reported better elsewhere.